### PR TITLE
Update gRPC docs for 2.29.0

### PR DESCRIPTION
**Pull request by JamesNK**
**Should not be merged** until 2.29.0 (non-preview) packages are on NuGet.

https://www.nuget.org/packages/Grpc.AspNetCore

### CODE REVIEW

File: aspnetcore/grpc/configuration.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ gRPC client configuration is set on `GrpcChannelOptions`. The following table de
...
-| HttpClient | `null` | The `HttpClient` used to make gRPC calls. This setting is an alternative to HttpHandler. |
-| DisposeHttpClient | `false` | If `true`, and an `HttpMessageHandler` or `HttpClient` is specified, then it will be disposed when the `GrpcChannel` is disposed. |
+| HttpHandler | New instance | The `HttpMessageHandler` used to make gRPC calls. A client can be set to configure a custom `HttpClientHandler` or add additional handlers to the HTTP pipeline for gRPC calls. If no `HttpMessageHandler` is specified, a new `HttpClientHandler` instance is created for the channel with automatic disposal. |
+| HttpClient | `null` | The `HttpClient` used to make gRPC calls. This setting is an alternative to `HttpHandler`. |
+| DisposeHttpClient | `false` | If set to `true` and an `HttpMessageHandler` or `HttpClient` is specified, the `HttpClient` is disposed when the `GrpcChannel` is disposed. |
```
Comment (JamesNK) [inline aspnetcore/grpc/configuration.md:45]: > the `HttpClient` is disposed when the `GrpcChannel` is disposed.

The setting now disposes either `HttpClient` or `HttpHandler`, depending on what was set. It is a little confusing but I didn't want to add another property for it. I don't know the best way to phrase that.

Comment (guardrex) [reply aspnetcore/grpc/configuration.md:45]: Oh, I see.

How about ...

> If set to \`true\` and an \`HttpMessageHandler\` or \`HttpClient\` is specified, then either the \`HttpHandler\` or \`HttpClient\` is disposed, respectively, when the \`GrpcChannel\` is disposed.

Comment (JamesNK) [reply aspnetcore/grpc/configuration.md:45]: Perfect

**Comment by guardrex** [comment]
Ok, all set. I'll merge when I see that the release packages have gone live.

**Pull request merged**